doc/templates.rst

```diff
@@ -664,6 +664,8 @@ string:
         {% if phone matches '/^[\\d\\.]+$/' %}
         {% endif %}
 
+    For simple string comparisons, the containment operator can be used.
```

Comment:
I would also add an example.

Reply:
There is an example is 12 lines down.

Reply:
Yeah I hesitated about that, but the fact that the containment operator section is just below would have seemed kinda duplicated to me. I also thought of an anchor but it didn't seem relevant to me either. But indeed nothing ensures that it won't move someday.

No problem if you'd like me to add one anyway :).